demo-client/src/main/java/com/illia/client/controller/DemoClientController.java

```diff
@@ -0,0 +1,42 @@
+package com.illia.client.controller;
+
+import com.illia.client.service.DemoService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.File;
+
+@Slf4j
+@RestController
+@RequestMapping("demo")
+public class DemoClientController {
+
+    @Autowired
+    private DemoService demoService;
+
+    @GetMapping("/uploadFile")
+    public String uploadFile(@RequestParam(name = "fileName") String fileName,
```

Comment:
Here it is better not to return String, but org.springframework.http.ResponseEntity, so you can handle actual http responses